cluster-scope/base/apiextensions.k8s.io/customresourcedefinitions/rayclusters.cluster.ray.io/customresourcedefinition.yaml

```diff
@@ -2,6 +2,7 @@ apiVersion: apiextensions.k8s.io/v1
 kind: CustomResourceDefinition
 metadata:
   name: rayclusters.cluster.ray.io
+  namespace: remote-sensing
 spec:
   group: cluster.ray.io
   scope: Namespaced
```

Comment:
@larsks CRD's can be namespaced as specified here. And because this is set and a namespace wasn't specified, this CRD ended up in some argocd default namespace (open-cluster-management I think, can't check right now since the cluster is down).

This document also says that CRDs can be namespaced. https://kubernetes.io/docs/tasks/extend-kubernetes/custom-resources/_print/#create-a-customresourcedefinition

The question is for @acemaster, since they provided us with this CRD with `scope` set to `Namespaced`. If this needs to be available clusterwide, then we will have to set `scope` to `Cluster`.